data/strings/strings.txt

```diff
@@ -4997,49 +4997,50 @@
-    hi = क्या आप वाकई जारी रखना चाहते हैं?
-    hu = Valóban folytatni akarja?
-    id = Apakah Anda yakin ingin melanjutkan?
-    it = Sei sicuro di voler continuare?
-    ja = 続行しますか？
-    ko = 계속하겠습니까?
-    lt = Tikrai norite tęsti?
-    mr = खात्रीपूर्वक पुढे जायचे?
-    nb = Er du sikker på at du ønsker å fortsette?
-    nl = Weet u zeker dat u wilt doorgaan?
-    pl = Kontynuować?
-    pt = Quer mesmo continuar?
-    pt-BR = Tem certeza que deseja continuar?
-    ro = Vrei să continui?
-    ru = Вы уверены, что хотите продолжить?
-    sk = Naozaj chcete pokračovať?
-    sv = Är du säker på att du vill fortsätta?
-    th = คุณแน่ใจหรือไม่ว่าคุณต้องการดำเนินการต่อ?
-    tr = Devam etmek istediğinizden emin misiniz?
-    uk = Продовжити?
-    vi = Bạn có chắc muốn tiếp tục không?
-    zh-Hans = 您确定要继续吗？
-    zh-Hant = 您確定要繼續嗎？
+    en = Are you sure you want to log out of your OpenStreetMap account?
+    af = Is jy seker jy wil by jou OpenStreetMap-rekening afmeld?
+    ar = هل أنت متأكد أنك تريد تسجيل الخروج من حساب OpenStreetMap الخاص بك؟
+    az = OpenStreetMap hesabınızdan çıxmaq istədiyinizə əminsiniz?
+    be = Вы ўпэўнены, што хочаце выйсці са свайго ўліковага запісу OpenStreetMap?
+    bg = Сигурни ли сте, че искате да излезете от профила си в OpenStreetMap?
+    ca = Esteu segur que voleu tancar la sessió del vostre compte d'OpenStreetMap?
+    cs = Opravdu se chcete odhlásit ze svého účtu OpenStreetMap?
+    da = Er du sikker på, at du vil logge ud af din OpenStreetMap-konto?
+    de = Sind Sie sicher, dass Sie sich von Ihrem OpenStreetMap-Konto abmelden möchten?
+    el = Είστε σίγουροι ότι θέλετε να αποσυνδεθείτε από το λογαριασμό σας στο OpenStreetMap;
+    es = ¿Está seguro de que desea salir de su cuenta de OpenStreetMap?
+    et = Kas olete kindel, et soovite oma OpenStreetMapi kontost välja logida?
+    eu = Ziur zure OpenStreetMap kontutik saioa amaitu nahi duzula?
+    fa = آیا مطمئن هستید که می خواهید از حساب OpenStreetMap خود خارج شوید؟
+    fi = Haluatko varmasti kirjautua ulos OpenStreetMap-tililtäsi?
+    fr = Êtes-vous sûr de vouloir vous déconnecter de votre compte OpenStreetMap ?
+    he = האם אתה בטוח שברצונך להתנתק מחשבון OpenStreetMap שלך?
+    hi = क्या आप वाकई अपने OpenStreetMap खाते से लॉग आउट करना चाहते हैं?
+    hu = Biztos, hogy ki akarsz jelentkezni az OpenStreetMap fiókodból?
+    id = Apakah Anda yakin ingin keluar dari akun OpenStreetMap Anda?
+    it = È sicuro di voler uscire dal suo account OpenStreetMap?
+    ja = 本当にOpenStreetMapアカウントからログアウトしますか？
+    ko = 오픈스트리트맵 계정에서 로그아웃하시겠습니까?
+    lt = Ar tikrai norite atsijungti nuo "OpenStreetMap" paskyros?
+    mr = तुमची खात्री आहे की तुम्ही तुमच्या OpenStreetMap खात्यातून लॉग आउट करू इच्छिता?
+    nb = Er du sikker på at du vil logge ut av OpenStreetMap-kontoen din?
+    nl = Weet u zeker dat u wilt uitloggen uit uw OpenStreetMap account?
+    pl = Czy na pewno chce się Pan/Pani wylogować ze swojego konta OpenStreetMap?
```

Comment:
Definitely too formal, should be:
"Czy na pewno chcesz się wylogować ze swojego konta OpenStreetMap?"